code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1317,11 +1317,12 @@
 
 /datum/reagent/stimulum
 	name = "Stimulum"
-	description = "An unstable experimental gas that greatly increases the energy of those that inhale it, while dealing increasing toxin damage over time."
+	description = "An unstable experimental gas that greatly increases the energy of those that inhale it, but also causes hypoxia"
 	reagent_state = GAS
 	metabolization_rate = REAGENTS_METABOLISM * 0.5 // Because stimulum/nitryl are handled through gas breathing, metabolism must be lower for breathcode to keep up
 	color = "E1A116"
 	taste_description = "sourness"
+	var/warn = 0
```

Comment:
variable name `warn` sounds like you'd like to warn a few times when conditions are met. I'd like to suggest `warned` instead, so that we can recognise the variable is only once notified.

Reply:
👍